src/pybind/mgr/orchestrator_cli/module.py

```diff
@@ -182,29 +196,82 @@ def _list_services(self, cmd):
                     s.version,
                     s.rados_config_location))
 
-            return HandleCommandResult(stdout="\n".join(lines))
+            return HandleCommandResult(odata="\n".join(lines))
 
-    def _service_add(self, cmd):
+    def _create_osd(self, inbuf, cmd):
+        """Create one or more OSDs
+
+        :cmd : Arguments for the create osd
+        """
+        #Obtain/validate parameters for the operation
+        cmdline_error = ""
+        if "drive_group" in cmd.keys():
+            params = cmd["drive_group"]
+        elif inbuf:
+            params = inbuf
+        else:
+            cmdline_error = "Please, use 'drive_group' parameter \
+                             or specify -i <input file>"
+
+        if cmdline_error:
+            return HandleCommandResult(-errno.EINVAL, rs=cmdline_error)
+
+        drive_group = orchestrator.DriveGroupSpec(params)
+
+        #Launch the operation in the orchestrator module
+        completion = self._oremote("create_osds", drive_group)
+
+        #Wait until the operation finishes
+        self._wait([completion])
+
+        #return result
+        return HandleCommandResult(odata=completion.result)
```

Comment:
```suggestion
        return HandleCommandResult(stdout=completion.result)
```

Reply:
Fixed, and reviewed all the calls to "HandleCommandResult" in this file to be sure that all uses the right "signature".